Countly.js

```diff
@@ -1585,8 +1585,9 @@ Countly.showStarRating = function (callback) {
  * @param {callback listener} [ratingWidgetCallback] This parameter is optional.
  */
 Countly.presentRatingWidgetWithID = function (widgetId, closeButtonText, ratingWidgetCallback) {
```

Comment:
in function's doc, return is missing if it returns smthng. Message is returned but, does it intended thing?